util/keccak-hasher/Cargo.toml

```diff
@@ -9,4 +9,5 @@ license = "GPL-3.0"
 ethereum-types = { version = "0.4", default-features = false, features = ["std", "serialize"] }
 tiny-keccak = "1.4.2"
 hashdb = { git = "https://github.com/artemii235/parity-common" }
-plain_hasher = { git = "https://github.com/artemii235/parity-common" }
\ No newline at end of file
+plain_hasher = { git = "https://github.com/artemii235/parity-common" }
```

Comment:
As `avoid using parity-common fork` is mentioned, I suppose it's worth trying to remove its usage here too.

Reply:
Missed this. I removed `keccak-hasher` at all. Please check